base/templates/base/base.html

```diff
@@ -1,129 +1,131 @@
+{% load static %}
+{% load bootstrap3 %}
 <!DOCTYPE html>
-<html>
-    
-    <head>
-        {% load bootstrap3 %}
-        {% bootstrap_css %}
-        {% bootstrap_javascript jquery=1%}
-        {% load staticfiles %}
-        {% block before-header %}{% endblock %}
-        <script type="text/javascript" src="{% static '/js/jsi18n'%}"></script>
-        <link rel="shortcut icon" href="{% static 'favicon.ico' %}"  type="image/x-icon" />
-        <title>
-            OEP{% block title %}{% endblock %}
-        </title>
-        <link rel="stylesheet" href="{% static 'css/theme-style.css' %}">
-        <link rel="stylesheet" href="{% static 'css/base-style.css' %}">
-        {% block after-header %}{% endblock %}
-    </head>
-
+<html lang="en">
+<head>
+    {% block before-head %}{% endblock before-head %}
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>OEP{% block title %}{% endblock title %}</title>
+    <link rel="shortcut icon" href="{% static 'favicon.ico' %}"  type="image/x-icon" />
+    {% bootstrap_css %}
+    <link rel="stylesheet" href="{% static 'css/theme-style.css' %}">
+    <link rel="stylesheet" href="{% static 'css/base-style.css' %}">
+    {% block after-head %}{% endblock after-head %}
+</head>
 <body>
```

Comment:
My block structure changes are mainly in the following `<body>` section. I divided the block structure into sections and sub-sections:

1.) The main sections are:

- header (formerly named "nav")
- main
- footer

2.) I kept the old block layout in sub-blocks and introduced a new naming convention for simplifying things, e.g.:

- main
    - main-content-body (formerly just called "content")

3.) I also introduced an additional left sidebar to the given right sidebar, which both are part of the main section, to not destroy the old layout structure (normally I would put the sidebars into an independent section):

- main
    - main-right-side-bar
    ...
    - main-left-side-bar